docs/UPDATE_CORE.md

```diff
@@ -15,10 +14,17 @@ Let's say the core version you want to upgrade to is `X.Y.Z`.
 1. `npm i deltachat-node@X.Y.Z @deltachat/jsonrpc-client@X.Y.Z`
 2. mention that you updated those dependencies in `CHANGELOG.md`
 
-If the core version is not uploaded to NPM yet, then you need to upload the files from our download server to NPM via `npm publish [url] --access=public` (if you have the rights to do so, otherwise ask us to do it).
-They are built by GitHub CI in the core repo and then uploaded to https://download.delta.chat/node/.
+If version `X.Y.Z` hasn't yet been published to `npm`, then you need to publish the files from our download server to `npm` via the following command:
 
-## Update to a development version (for a pull request that needs a new core to be tested)
+```sh
+npm publish [url] --access=public
```

Comment:
I'd like to insert the actual url here instead of a placeholder. I'm _guessing_ that's `https://download.delta.chat/node`. Is this correct?

Reply:
Oh nvm. So the real url should point to the corresponding `.tar.gz` file right?

Reply:
yes